datafusion/core/src/physical_optimizer/sort_enforcement.rs

```diff
@@ -1101,22 +1064,21 @@ mod tests {
             sort,
         );
 
-        // let filter_exec = sort_exec;
-        let physical_plan = window_exec("non_nullable_col", sort_exprs, filter);
+        let physical_plan = bounded_window_exec("non_nullable_col", sort_exprs, filter);
```

Comment:
I have changed the `window_exec` with `bounded_window_exec` because the window expression used in the expression can run with `BoundedWindowAggExec` (If it were come from sql query it would work with `BoundedWindowExec`). I just want it to be consistent with general behaviour.